### ISSUE TITLE
install internal binaries in /usr/lib/vpn-server-node/bin ?

### ISSUE THREAD

**Issue by gijzelaerr** [eduvpn/vpn-server-node#16]
lintian complains:
```
W: vpn-server-node: non-standard-dir-in-usr usr/libexec/
W: vpn-server-node: file-in-unusual-dir usr/libexec/vpn-server-node-client-connect
W: vpn-server-node: file-in-unusual-dir usr/libexec/vpn-server-node-client-disconnect
W: vpn-server-node: file-in-unusual-dir usr/libexec/vpn-server-node-verify-otp
```
For debian this should probably be `/usr/lib/vpn-server-node/bin`. The prefix is hardcoded in `src/OpenVpn.php`. I can patch this, but maybe better to synchronise this?

**Comment by fkooman** [comment]
https://fedoraproject.org/wiki/Packaging:Guidelines#Libexecdir

It looks like I should at least move the files to `/usr/libexec/vpn-server-node`...

**Comment by fkooman** [comment]
I guess, if on Debian those files really should go somewhere else I'll make it configurable...

**Comment by fkooman** [comment]
`/usr/lib/vpn-server-node/bin` vs `/usr/libexec/vpn-server-node/`

**Comment by fkooman** [comment]
There is a "small" problem when we try to fix this for existing deploys, after package upgrade all instances need to regenerate their configuration and restart OpenVPN. Unless we fix it and temporary keep symlinks in place to the new location. So, maybe this is something we'd have to fix later?

**Comment by fkooman** [comment]
e8c5e1e800702651ea6990fcd9dbc963e899b420

**Comment by gijzelaerr** [comment]
if you ask me it is not critical, it is just a warning and doesn't look pretty on the Debian filesystem since libexec is never used.

**Comment by fkooman** [comment]
Alright, for 1.0.4 I fixed this. By default the path is `/usr/libexec/vpn-server-node`, but you can patch it for Debian to become `/usr/lib/vpn-server-node/bin` in `src/OpenVpn.php`.

**Comment by fkooman** [comment]
On Fedora/CentOS I'll keep symlinks from the old location to be able to work with existing OpenVPN server configuration files.

**Comment by fkooman** [comment]
Will be fixed in 1.0.4
